fvm/src/gas/price_list.rs

```diff
@@ -219,7 +219,7 @@ lazy_static! {
 
         block_memory_retention: ScalingCost {
             flat: Gas::zero(),
-            scale: Gas::new(8),
+            scale: Gas::new(10),
```

Comment:
This looks like an increase, but it's not. This fee now represents the _minimum_ per/byte gas fee, not an additional charge.

Reply:
Should we copy the reasoning in the PR body to rustdoc comments on the struct field definition, so that it's in-band with the code?